phpBB/includes/acp/acp_profile.php

```diff
@@ -161,6 +164,11 @@ function main($id, $mode)
 
 			case 'activate':
 
+				if (!check_link_hash($request->variable('hash', ''), 'acp_profile'))
```

Comment:
I can't find in the style file (acp_profile.html) where the variable hash is set?  Is it set somewhere else?  I am getting submitted form was invalid when trying to edit a profile field.